mutatex/utils.py

```diff
@@ -86,21 +87,31 @@ def init_arguments(arguments, parser):
 
     return parser
 
-def get_font_list(str=True):
-
-    flist = matplotlib.font_manager.get_fontconfig_fonts()
-    names = [ matplotlib.font_manager.FontProperties(fname=fname).get_name() for fname in flist ]
-    if not str:
+def get_font_list(return_as_string=True):
+    try:
+        flist = matplotlib.font_manager.findSystemFonts()
+    except AttributeError:
+        flist = matplotlib.font_manager._get_fontconfig_fonts()
+    names = []
+    for fname in flist:
+        try:
+            name = matplotlib.font_manager.FontProperties(fname=fname).get_name()
+            names.append(name)
+        except:
+            pass
+    if type(names) is not str:
         return names
     return textwrap.fill(", ".join(sorted(list(set(names)))), width=69)
```

Comment:
here it looks like we're not using the `return_as_string` argument as we were before the argument name change. It should probably be something like:

```
 def get_font_list(return_as_string=True):
    ....
    if not return_as_string:
        return names
    return ...
```